Sources/MapboxNavigation/NavigationMapView+Annotations.swift

```diff
@@ -35,15 +35,27 @@ extension NavigationMapView {
         guard style.image(withId: "RouteInfoAnnotationLeftHanded") == nil,
               style.image(withId: "RouteInfoAnnotationRightHanded") == nil else { return }
         
+        let bottomInset = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
+        
         // Right-hand pin
         if let image = Bundle.mapboxNavigation.image(named: "RouteInfoAnnotationRightHanded") {
             // define the "stretchable" areas in the image that will be fitted to the text label
             // These numbers are the pixel offsets into the PDF image asset
-            let stretchX = [ImageStretches(first: Float(33), second: Float(52))]
-            let stretchY = [ImageStretches(first: Float(32), second: Float(35))]
+            var stretchX: [ImageStretches]!
```

Comment:
The `stretchX` and `stretchY` don't change with the `bottomInset`. So we should keep them, and only update the `imageContent` with ternary operator based on the  `bottomInset`.  And the default `ImageContent(left: 28, top: 24, right: 58, bottom: 40)` is constant for two cases as well.